src/ucp/wireup/select.c

```diff
@@ -397,6 +398,7 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
     ucp_context_h context                         = worker->context;
     ucp_wireup_select_info_t sinfo                = {0};
     int found                                     = 0;
+    char *wireup_info;
```

Comment:
move it after local_iface_flags since initialized vars should before un-initialized ones